packages/test/test-end-to-end-tests/src/test/gc/gcTombstoneDataStores.spec.ts

```diff
@@ -767,7 +767,7 @@ describeCompat("GC data store tombstone tests", "NoCompat", (getTestObjectProvid
 					clientType: "noninteractive/summarizer",
 				},
 				{
-					eventName: "fluid:telemetry:Summarizer:Running:SweepReadyObject_Revived",
+					eventName: "fluid:telemetry:Summarizer:Running:TombstoneReadyObject_Revived",
```

Comment:
Not as part of this PR but we should get rid of this and sweep ready errors. We already have the tombstone error logs.

Reply:
Noted on my list of follow-ups.  I think in the past we considered this but for some reason kept them.